docs/readmes/lte/deploy_install.md

```diff
@@ -37,15 +35,15 @@ satisfies the following requirements:
 
 ### 1. Create boot USB stick and install Ubuntu on your AGW host
 
-- Download the Ubuntu Server 20.04 LTS `.iso` image from the Ubuntu website
+- Download the Ubuntu Server 20.04 LTS `.iso` image from the Ubuntu website. Verify its integrity by checking [the hash](https://releases.ubuntu.com/20.04/SHA256SUMS).
 - Create a bootable USB using this [Etcher tutorial](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos#0)
 - Boot your AGW host from USB
     - Press F11 to select boot sequence. WARNING: This might be different for your machine.
     - If you see two options to boot from USB, select the non-UEFI option.
 - Install and configure your Access Gateway according to your network defaults.
     - Make sure to enable ssh server and utilities (untick every other).
 - Connect your SGi interface to the internet and select this port during the
-installation process to get an IP using DHCP.
+installation process to get an IP using DHCP. (Consider enabling DHCP snooping to mitigate [Rogue DHCP](https://en.wikipedia.org/wiki/Rogue_DHCP))
```

Comment:
How does one enable this in practice? It would be nice to give more specific instructions.

Reply:
We could have the text "DHCP snooping" link to https://en.wikipedia.org/wiki/DHCP_snooping, like [DHCP snooping](https://en.wikipedia.org/wiki/DHCP_snooping).

Reply:
That sounds good, thanks.

Reply:
This isn't a blocker so I will go ahead and approve this for now.